modules/ROOT/nav.adoc

```diff
@@ -8,5 +8,6 @@
 ** xref:decision-points-service.adoc[Service]
 ** xref:decision-points-embedded.adoc[Embedded]
 * xref:audit-log-collection.adoc[Audit log collection]
+* xref:access-controls.adoc[Access controls]
```

Comment:
This doesn't really fit here. There should be a dedicated section called "Managing Cerbos Hub" or "Administration" and this should go under "Managing access to users" or something like that.